scapy/layers/tls/session.py

```diff
@@ -412,7 +413,7 @@ def __init__(self,
         self.advertised_tls_version = 0x0303
 
         # The agreed-upon TLS version found in the ServerHello.
-        self.tls_version = None
+        self.tls_version = 0
```

Comment:
This will fix the `None > 0x0303` Bug.

Reply:
Sorry, but I think I prefer the first way you had fixed it. `None` has a special meaning. @mtury what do you think?

Reply:
I could set -1 if it's clearer

Reply:
I don't think it's clearer. `None` is clearer. The problem is you compare a value, with a "non-value". If you use `0` (or even `-1`), that looks like a value, and you use the hack that your non-value compares with a value (that's what happened with `None` compared to integers in Python 2).